config/routes.rb

```diff
@@ -8,6 +8,11 @@
   root "home#index"
 
   devise_for :users
+  devise_for :teachers, skip: [:registrations]
```

Comment:
Let's remove the devise for teachers. 
It will create a new route for teachers to use a different login and we don't want that.  And Teachers are users so this will cause conflicts with devise as well.  We should be fine with just devise for users.